src/containers/Admin/AdminContainter.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import { authAccessToken } from "../../utils/auth";
+import { AdminLogin } from "../../components/Admin/AdminLogin/AdminLogin";
+import { AdminPopUpClinic } from "../../components/Admin/AddPopUpClinic/AdminPopUpClinic";
+
+export function AdminContainer() {
+  const [accessToken, setAccessToken] = useState(
+    window.sessionStorage.getItem("bearerToken") || null,
+  );
+
+  if (accessToken && !authAccessToken(accessToken)) {
```

Comment:
Looks like we're hitting the api on every render so long as we have an `accessToken` in state. We may want to check the previous value of `accessToken` and only hit the api if there has been a change. To compare previous and current values, you can use this hook: https://usehooks.com/usePrevious/